install/operator/pkg/syndesis/configuration/configuration.go

```diff
@@ -297,6 +306,45 @@ func GetAddons(configuration Config) []AddonInstance {
 	}
 }
 
+/*
+ * For testing if the giving platform is Openshift
+ */
+func ApiCapabilities(apiClient kubernetes.Interface) (*ApiServerSpec, error) {
```

Comment:
I would argue that this is the right place for this function

Reply:
ok?

Reply:
Moved ApiCapabilities to its own package.